### DISCUSSION TITLE
Where is the example on converting a contract on our file system?

### DISCUSSION THREAD

**Discussion by AlphaDio** [wevm/wagmi#4135] [General]
https://wagmi.sh/cli/getting-started

Why is there an example of getting an ABI from etherscan, when one of the typical use cases is going to be to convert contracts built on our own machines???

Either I'm reading this wrong, or there is no native way to convert a contract locally, instead I have to deploy on Ethereum and THEN I can get the ts ABI from it??? I really hope I'm just in dev fatigue mode or reading it wrong.

**Comment by tmm** [comment]
https://wagmi.sh/cli/config/configuring-cli#async-config
